src/Components/Web.Extensions/src/InputFile/RemoteBrowserFileStreamOptions.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Components.Web.Extensions
+{
+    /// <summary>
+    /// Repesents configurable options for <see cref="RemoteBrowserFileStream"/>.
+    /// </summary>
+    public class RemoteBrowserFileStreamOptions
+    {
+        /// <summary>
+        /// Gets or sets the maximum segment size for file data sent over a SignalR circuit.
+        /// This only has an effect when using Blazor Server.
+        /// </summary>
+        public int SegmentSize { get; set; } = 20 * 1024; // SignalR limit is 32K.
```

Comment:
Could we document these defaults as part of the doc comment? I'd be fine doing this in a follow up PR